lib/backend/k8s/conversion/conversion.go

```diff
@@ -173,7 +173,7 @@ func (c Converter) PodToWorkloadEndpoint(pod *kapiv1.Pod) (*model.KVPair, error)
 
 	// Generate the interface name based on workload.  This must match
 	// the host-side veth configured by the CNI plugin.
-	interfaceName := VethNameForWorkload(wepName)
+	interfaceName := VethNameForWorkload(wepName + pod.Namespace)
```

Comment:
For paranoia sake - better to join these together with, say,  a `/` separator

Reply:
hmm ok, not sure why it's necessary but I feel the same way now that you pointed it out

Reply:
i don't think it'll cause issues with the current code because of the fact that the endpoint is always eth0

but suppose in the future we want `eth1` on namespace `0x` and `eth10` on namespace `x`.  Adds up to the same thing.